pytorch_lightning/accelerators/ddp_cpu_spawn_backend.py

```diff
@@ -11,14 +11,191 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License
-from pytorch_lightning.accelerators.ddp_spawn_backend import DDPSpawnBackend
+import os
+import re
 
+import torch
+import torch.distributed as torch_distrib
+import torch.distributed as dist
+import torch.multiprocessing as mp
 
-class DDPCPUSpawnBackend(DDPSpawnBackend):
+from pytorch_lightning import _logger as log
+from pytorch_lightning.accelerators.base_backend import Accelerator
+from pytorch_lightning.utilities import AMPType
+from pytorch_lightning.utilities.cloud_io import atomic_save
+from pytorch_lightning.utilities.distributed import rank_zero_only, rank_zero_warn
+from pytorch_lightning.utilities.distributed import find_free_network_port
 
-    def model_to_device(self, model, process_idx, is_master):
-        pass
+try:
+    from hydra.core.hydra_config import HydraConfig
+    from hydra.utils import get_original_cwd, to_absolute_path
+except ImportError:
+    HYDRA_AVAILABLE = False
+else:
+    HYDRA_AVAILABLE = True
 
-    def get_device_ids(self):
+
+class DDPCPUSpawnBackend(Accelerator):
```

Comment:
naming nit: if run with torchelastic, then torchelastic does the spawn, not the backend